contrib/debian/control

```diff
@@ -48,11 +48,9 @@ Pre-Depends: adduser,
              dpkg (>= 1.17.14),
              libcap2-bin (>=1:2.0),
              lsb-base (>= 3.1-23.2)
-Conflicts: netdata-core,
-           netdata-plugins-bash,
+Conflicts: netdata-plugins-bash,
```

Comment:
We need to keep `netdata-core` here. It’s part of the packages provided by Debian/Ubuntu for Netdata, not anything we ever provided, and we _do_ conflict with it.